src/application.js

```diff
@@ -5,11 +5,53 @@
 Marionette.Application = Marionette.Object.extend({
   cidPrefix: 'mna',
 
+  constructor: function(options) {
+    options = options || {};
+
+    this._initRegion(options);
+
+    Marionette.Object.prototype.constructor.apply(this, arguments);
+  },
+
+  _initRegion: function(options) {
+    var region = options.region || this.region;
+
+    if (_.isString(region)) {
+      this._region = new Marionette.Region({
+        el: region
+      });
+      return;
+    }
+
+    this._region = region;
+  },
+
+  getRegion: function() {
+    return this._region;
+  },
+
+  showView: function(view) {
+    this.getRegion().show(view);
+  },
+
+  isRunning: function() {
+    return !!this._isRunning;
+  },
```

Comment:
Why have this? We should get rid of this as well as `start` and `stop`, unless we want to hide the router configuration behind `start`, like...

```js
MyApp.start({
  html5: true
});
```